Modules/Quest/AvailableQuests/AvailableQuests.lua

```diff
@@ -64,18 +64,56 @@ end
 function AvailableQuests.DrawAvailableQuest(quest) -- prevent recursion
     --? Some quests can be started by both an NPC and a GameObject
 
+    local added = 0
+    if quest.Starts.Item then
+        local items = quest.Starts.Item
+        for i = 1, #items do
+            local item = QuestieDB:GetItem(quest.Starts.Item[i])
+
+            if (not item) then
+                -- TODO: This check can be removed once the DB is fixed
+                Questie:Error("Item not found for quest", quest.Id, "Item ID:", items[i], "- Please report this on Github or Discord!")
+                return
+            end
+
+            if item.npcDrops then
+                for _, npc in ipairs(item.npcDrops) do
+                    local no = QuestieDB:GetNPC(npc)
+                    if Questie.db.profile.availableIconLimit == 0 or added < Questie.db.profile.availableIconLimit then
+                        added = added + _AddStarter(no, quest, "im_"..npc, (Questie.db.profile.availableIconLimit == 0 and 0) or (Questie.db.profile.availableIconLimit - added))
```

Comment:
I think it's easier to read when the value for the `limit` param is calculated in a separated line. Same for the other places below.